### ISSUE TITLE
Add Google Analytics tag to Adventist tenants

### ISSUE THREAD

**Issue by jillpe** [scientist-softserv/adventist_knapsack#557]
We need help to add tags to our sites so we can track users via Google Analytics. Since we intend to publicly share the site links in late June, we'd like to enable Google Analytics beforehand, so we have data from "day one." 

Instructions we received from Google: "To enable data collection, you need to add your new Google Analytics tag (gtag.js) to the header of every page of your site. You'll find it here. Once you take this step, your new property will start populating with data. Below is the Google tag for this account. Copy and paste it in the code of every page of your website, immediately after the <head> element. Don’t add more than one Google tag to each page. "

We can provide the tag in a later edit of this ticket. 

- [ ] Get GA tag from Katharine

## Acceptance
- [ ] GA Tag added to ADL Hyku
- [ ] If GA isn't working by adding Tag, go back to Katharine and/or create new ticket

**Comment by jillpe** [comment]
This is in core hyku. To Do:
- set up accounts/configurations from both sides (1)

**Comment by ShanaLMoore** [comment]
blocked by https://github.com/notch8/palni_palci_knapsack/issues/314

**Comment by fabio-adl** [comment]
<img width="1499" height="769" alt="Image" src="https://github.com/user-attachments/assets/4148a563-a9c8-4ee2-ae78-326b892c91d6" />

<img width="1491" height="972" alt="Image" src="https://github.com/user-attachments/assets/64d554e5-e488-497d-876c-2e806f1dd94a" />

The analytics is working, as you can see in the snapshot. I tested for two instances. But I get this error when I try to connect to the dashboard.

**Comment by fabio-adl** [comment]
Yes, you should proceed to prod.
